### DISCUSSION TITLE
Using with React Native

### DISCUSSION THREAD

**Discussion by k6jge3jf10o7** [pmndrs/use-cannon#428] [Q&A]
Is it possible to use use-cannon with react-three-fiber in React Native project? Is there any example how to do this?

I can fix first two problems:
- Problems resolving module
- Missing property atob

But I have not managed to continue from "Property 'Worker'doesn't exist.

**Comment by Deboracgs** [comment]
I want to use react-native, but I'm receiving the error the module doesn't exists, even been installed.

**Comment by isaac-mason** [comment]
@react-three/cannon isn't supported in react native as it uses web workers.

See this discussion for an example of how to use cannon-es directly with r3f: https://github.com/pmndrs/use-cannon/issues/63#issuecomment-1501493798
